src/strategies/unigraph/index.ts

```diff
@@ -0,0 +1,66 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const UNISWAP_SUBGRAPH_URL = {
+  1: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2',
+};
+
+export async function strategy(network, provider, addresses, options, snapshot) {
+  const params = {
+    users: {
+      __args: {
+        where: {
+          id_in: addresses.map(address => address.toLowerCase())
+        },
+        first: 10,
+      },
+      id : true,
+      liquidityPositions : {
+        __args : {
+          where : {
+            liquidityTokenBalance_gt : 0
+          }
+        },
+        liquidityTokenBalance  : true,
+        pair : {
+          id : true,
+          token0 : {
+            id : true
+          },
+          reserve0 : true,
+          token1 : {
+            id : true
+          },
+          reserve1 : true,
+          totalSupply : true
+        }  
+      }
+    }
+  };
+  if (snapshot !== 'latest') {
+    // @ts-ignore
+    params.poolShares.__args.block = { number: snapshot };
+  }
+  const tokenAddress = options.address.toLowerCase();
+  const result = await subgraphRequest(UNISWAP_SUBGRAPH_URL[network], params);
+  const score = {};
+  if (result && result.users) {
+    result.users.forEach(u => {
+      u.liquidityPositions.filter(lp => 
+        lp.pair.token0.id == tokenAddress 
+        || lp.pair.token1.id == tokenAddress)
+                          .forEach(lp => {
```

Comment:
I can see the space intend is wrong here, can you run the lint script? That should fix that. You can use `npm run lint`